resourceDescriptors.yaml

```diff
@@ -1,6 +1,16 @@
 # AGR Schema Resource Descriptors YAML.
 # When submitting identifiers, please use the appropriate example_id/gid field listed below.
 
+#
+# AGR
+#
+
+- db_prefix: Alliance of Genome Resources
```

Comment:
@markquintontulloch - at some point this db_prefix should be something without spaces, etc. :)